### ISSUE TITLE
Simplify management of overpayments from customers and display credit notes balance in a separate line of balance sheet.

### ISSUE THREAD

**Issue by tranx** [bigcapitalhq/bigcapital#464]
We sometime receive money from customers that pay us before their invoice is created.

The reception of these over-payments is currently a bit annoying to manage currently on bigcapital, what we do is that we create an invoice, then transfer it to a credit note, and then apply it to the invoice when the correct invoice gets created. It is not impossible,but it is a bit annoying to have to create an invoice to create the payment to then converting in a credit note, it would be nicer to be able to create a credit note directly for payment received that exceeds the amount of existing invoice.

There may be other ways to do this in bigcapital, we currently thing using credit note is the most natural way as it seems the most integrated way in bigcapital (Though credit note does not seem to appear explicitly as a separate line in the balance sheet - it would be nicer to have amount of unpaid invoices, and amount of credit notes as two separate lines (as one is debtor and the other one is creditor) currently they are merged in a single line). 
There is finally a question of knowing if "credit notes" should be displayed in the "aging payables" reports which is currently limited to vendors.

**Comment by abouolia** [comment]
I would inform that feature is live in the latest update. check it out. @tranx

**Comment by tranx** [comment]
Hi Ahmed,

I have upgraded. Trying to have a look.
I am having issues when restarting the docker container. I may miss
something on the config migration.
I have disabled new_relic log by setting
   - NEW_RELIC_ENABLED: "false"
Which from what I understand is the reply to
https://github.com/bigcapitalhq/bigcapital/issues/507


But I keep landing on the page for taking a subscription and my bigcapital
does not switch to my tenant database.
Observation from mariadb shows that last query to database was :
select `TENANTS_METADATA`.* from `TENANTS_METADATA` where
`TENANTS_METADATA`.`TENANT_ID` in (1)

I can't see any errors in logs on the server - db seems not to have changed.
Have you got any idea of the issue I may be facing ? I will keep trying to
find what I may need to add -

Bertrand
